website/docs/r/synthetics_script_monitor.html.markdown

```diff
@@ -170,6 +170,8 @@ resource "newrelic_synthetics_script_monitor" "monitor" {
   runtime_type_version = "100"
   runtime_type         = "CHROME_BROWSER"
   script_language      = "JAVASCRIPT"
+  device_type          = "MOBILE"
+  device_orientation   = "LANDSCAPE"
```

Comment:
@gmanandhar-nr you can also add these attributes to the simple browser monitor (`newrelic_synthetics_monitor` resource with type `BROWSER`) too.